akka-actor/src/main/scala/akka/io/SimpleDnsCache.scala

```diff
@@ -36,10 +37,10 @@ class SimpleDnsCache extends Dns with PeriodicCacheCleanup {
   }
 
```

Comment:
`class SimpleDnsCache` can't be moved to another package if that was the question.
It's public api (not sure why it's public but that is how it is).